bundles/gcp-bundle/build.gradle.kts

```diff
@@ -49,6 +49,7 @@ tasks.withType(ShadowJar::class.java) {
   relocate("org.apache.httpcomponents", "org.apache.gravitino.shaded.org.apache.httpcomponents")
   relocate("org.apache.commons", "org.apache.gravitino.shaded.org.apache.commons")
   relocate("com.google", "org.apache.gravitino.shaded.com.google")
+  relocate("com.fasterxml", "org.apache.gravitino.shaded.com.fasterxml")
```

Comment:
Just curious, why do we need to relocate this one?

Reply:
It conflict with the packages in IcebergRESTServer when start up in docker image.

Reply:
Does this happen when we run `IcebergRESTServer` without docker image?

Reply:
no